### PR TITLE
Update translations in setup

### LINKED ISSUE #1518: Updating I18N datas
Hi guys, I'm totally new to Thelia, and found many typos in the different texts.
 
Where do I have to update this all to make a PR? Because I saw that many texts are present in both the `setup/insert.sql` file and in the `setup/I18n/*.php` files. Do I have to update both?

### PR DESCRIPTION
**Pull request by Pierstoval**
Many expressions were incorrect in french. I also standardized some things, like "JavaScript or javascript", to be sure the word is always the same in the whole app.
 
First PR, don't know Thelia that much, hope it fits/works/helps :+1: 

Will fix #1518

### CODE REVIEW

File: setup/I18n/fr_FR.php [hunk 1]
```diff
@@ -6,23 +6,23 @@
...
-    'Address creation - at the top of the form' => 'Création adresse - en haut du formulaire',
-    'Address update - CSS stylesheet' => 'Modification adresse - feuille de style CSS',
-    'Address update - after javascript include' => 'Modification adresse - après l\'inclusion des javascript',
-    'Address update - after javascript initialisation' => 'Modification adresse - après l\'initialisation du javascript',
-    'Address update - at the bottom' => 'Modification adresse - en bas',
-    'Address update - at the bottom of the form' => 'Modification adresse - en bas du formulaire',
-    'Address update - at the top' => 'Modification adresse - en haut',
-    'Address update - at the top of the form' => 'Modification adresse - en haut du formulaire',
+    'Address creation - CSS stylesheet' => 'Création d\'adresse - feuille de style CSS',
+    'Address creation - after javascript include' => 'Création adresse - après l\'inclusion du JavaScript',
```
Comment (lunika) [inline setup/I18n/fr_FR.php]: you put ```création d'adresse``` in insert.sql file

File: setup/I18n/fr_FR.php [hunk 2]
```diff
@@ -6,23 +6,23 @@
...
+    'Address creation - after javascript initialisation' => 'Création adresse - après l\'initialisation du JavaScript',
```
Comment (lunika) [inline setup/I18n/fr_FR.php]: you put création d'adresse in insert.sql file

File: setup/insert.sql [hunk 3]
```diff
@@ -2925,69 +2925,69 @@ INSERT INTO `hook_i18n` (`id`, `locale`, `title`, `chapo`, `description`) VALUES
...
     (73, 'fr_FR', 'Page contact - en cas de succès', NULL, NULL),
     (74, 'fr_FR', 'Page newletter - en haut', NULL, NULL),
     (75, 'fr_FR', 'Page newletter - en bas', NULL, NULL),
     (76, 'fr_FR', 'Page newletter - feuille de style CSS', NULL, NULL),
-    (77, 'fr_FR', 'Page newletter - après l\'inclusion des javascript', NULL, NULL),
-    (78, 'fr_FR', 'Page newletter - après l\'initialisation du javascript', NULL, NULL),
-    (79, 'fr_FR', 'Echec du paiement - feuille de style CSS', NULL, NULL),
-    (80, 'fr_FR', 'Echec du paiement - après l\'inclusion des javascript', NULL, NULL),
-    (81, 'fr_FR', 'Echec du paiement - initialisation du javascript', NULL, NULL),
+    (77, 'fr_FR', 'Page newletter - après l\'inclusion du JavaScript', NULL, NULL),
```
Comment (lunika) [inline setup/insert.sql]: you put ```page newsletter``` in fr_FR.php file

File: setup/insert.sql [hunk 4]
```diff
@@ -2925,69 +2925,69 @@ INSERT INTO `hook_i18n` (`id`, `locale`, `title`, `chapo`, `description`) VALUES
...
+    (78, 'fr_FR', 'Page newletter - après l\'initialisation du JavaScript', NULL, NULL),
```
Comment (lunika) [inline setup/insert.sql]: you put ```page newsletter``` in fr_FR.php file

File: setup/insert.sql [hunk 5]
```diff
@@ -2925,69 +2925,69 @@ INSERT INTO `hook_i18n` (`id`, `locale`, `title`, `chapo`, `description`) VALUES
...
+    (67, 'fr_FR', 'Mot de passe perdu - initialisation du JavaScript', NULL, NULL),
     (68, 'fr_FR', 'Page du choix du langage - en haut', NULL, NULL),
     (69, 'fr_FR', 'Page du choix du langage - en bas', NULL, NULL),
     (70, 'fr_FR', 'Page du choix du langage - feuille de style CSS', NULL, NULL),
-    (71, 'fr_FR', 'Page du choix du langage - après l\'inclusion des javascript', NULL, NULL),
-    (72, 'fr_FR', 'Page du choix du langage - après l\'initialisation du javascript', NULL, NULL),
+    (71, 'fr_FR', 'Page du choix du langage - après l\'inclusion du JavaScript', NULL, NULL),
+    (72, 'fr_FR', 'Page du choix du langage - après l\'initialisation du JavaScript', NULL, NULL),
     (73, 'fr_FR', 'Page contact - en cas de succès', NULL, NULL),
     (74, 'fr_FR', 'Page newletter - en haut', NULL, NULL),
```
Comment (lunika) [inline setup/insert.sql]: you put ```page newsletter``` in fr_FR.php file

File: setup/insert.sql [hunk 6]
```diff
@@ -2925,69 +2925,69 @@ INSERT INTO `hook_i18n` (`id`, `locale`, `title`, `chapo`, `description`) VALUES
...
     (75, 'fr_FR', 'Page newletter - en bas', NULL, NULL),
```
Comment (lunika) [inline setup/insert.sql]: you put ```page newsletter``` in fr_FR.php file

File: setup/insert.sql [hunk 7]
```diff
@@ -2925,69 +2925,69 @@ INSERT INTO `hook_i18n` (`id`, `locale`, `title`, `chapo`, `description`) VALUES
...
     (76, 'fr_FR', 'Page newletter - feuille de style CSS', NULL, NULL),
```
Comment (lunika) [inline setup/insert.sql]: you put ```page newsletter``` in fr_FR.php file

File: setup/I18n/fr_FR.php [hunk 8]
```diff
@@ -6,23 +6,23 @@
...
+    'Address creation - at the bottom' => 'Création d\'adresse - en bas',
+    'Address creation - at the bottom of the form' => 'Création d\'adresse - en bas du formulaire',
+    'Address creation - at the top' => 'Création d\'adresse - en haut',
+    'Address creation - at the top of the form' => 'Création d\'adresse - en haut du formulaire',
+    'Address update - CSS stylesheet' => 'Modification d\'adresse - feuille de style CSS',
+    'Address update - after javascript include' => 'Modification adresse - après l\'inclusion du JavaScript',
```
Comment (lunika) [inline setup/I18n/fr_FR.php]: you put ```Modification d'adresse``` in insert.sql file

**Comment by Pierstoval** [comment]
Fixed all inconsistencies, thanks @lunika for your feedback! :+1:

**Comment by lunika** [comment]
Thanks

**Pull request merged**